packages/ndla-ui/src/RadioButtonGroup/component.radio-button-group.scss

```diff
@@ -12,7 +12,7 @@
     font-weight: 600;
   }
   &__label {
-    @include font-size(16px, 28px);
+    @include font-size(16px, 28px, $important: true);
```

Comment:
Stylingen i form.css i editorial blir satt for _alle_ labels og input-felter. Så derfor bruker jeg `!important` her.

Optimalt sett så skulle vel stylingen i ed vært definert på en klasse eller lignende, men er litt redd for å endre den og ødelegge noe ettersom den er satt for _alt_.
(Kom gjerne med en mening her :smile: )

Reply:
Vil ikke denne alltid overskrive en generell label, når denne har spesifikk klasse?

Reply:
Kansje du kan endre
`&__label {`
om til 
`label.c-radio-button-group__label {`
Da burde den vertfall overskrive en vanlig
`label {`

Reply:
Egentlig hvorfor skriver vi ikke om all cssen til vanlig sass?

Reply:
> Vil ikke denne alltid overskrive en generell label, når denne har spesifikk klasse?

Jeg hadde håpt det, men det er ikke tilfellet.

> Kansje du kan endre
> `&__label {`
> om til
> `label.c-radio-button-group__label {`
> Da burde den vertfall overskrive en vanlig
> `label {`

Yap det fungerer så vidt jeg kan se. 
Jeg gjør det :+1: 
Føles litt småskittent og ikke bruke parent-selectoren når den finnes(?), men i guess det er bedre enn `!important`

> Egentlig hvorfor skriver vi ikke om all cssen til vanlig sass?

Den som er vanlig css i ed tenker du på, eller _all_ css'en? Liker du sass bedre enn css-in-js?

---


Vet du hvordan vi får med scss når vi gjør linking og hot-reloading?
Når jeg tester dette så må jeg bygge frontend-packages på nytt og restarte editorial dev-serveren for å få endret css'en.
Blir ganske lang roundtrip :sweat_smile: